static/js/controllers/reports.js

```diff
@@ -131,34 +188,65 @@ var _ = require('underscore'),
           return DB.get()
             .get(report)
             .then(FormatDataRecord);
-        } else {
-          return FormatDataRecord(report);
         }
+        return FormatDataRecord(report);
       };
 
       $scope.refreshReportSilently = function(report) {
-        _fetchFormattedReport(report)
+        return _fetchFormattedReport(report)
           .then(function(doc) {
-              _setSelected(doc[0]);
-            })
+            _setSelected(doc[0]);
+          })
           .catch(function(err) {
             $log.error('Error fetching formatted report', err);
           });
       };
 
+      var spliceSelected = function(id) {
```

Comment:
also `getNextHalfHour` -> `_getNextHalfHour`, `initEditMessageModal`, `deselectAll`, `setTitle`, `getFields`... 
Unless I'm not getting the notation.